packages/voila/src/manager.ts

```diff
@@ -42,7 +42,7 @@ export class WidgetManager extends KernelWidgetManager {
       {
         safe: false,
         mimeTypes: [WIDGET_MIMETYPE],
-        createRenderer: options => new WidgetRenderer(options)
+        createRenderer: options => new WidgetRenderer(options, this as any)
```

Comment:
Using `as any` because our WidgetManager does not provide all the needed APIs. Would that be a problem @jtpio ? 

At least it's not a problem for the WidgetRenderer itself.

Reply:
I guess it might, but worth trying it out.

Wondering whether the upstream typings could use `LabWidgetManager` instead, similar to https://github.com/jupyter-widgets/ipywidgets/pull/3561/files.